src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -147,8 +148,22 @@ def create_task_mappings(
                     changed_files = _get_filtered_files(diff, file_regex, repo_name)
 
                     if module_name:
-                        cur_module = _get_associated_module(version, module_name)
-                        prev_module = _get_associated_module(prev_version, module_name)
+                        try:
+                            cur_module = _get_associated_module(version, module_name)
+                            prev_module = _get_associated_module(prev_version, module_name)
+
+                            # even though we don't need the module info for next_version, we run
+                            # this check to raise an error if the next version has a config error
+                            _get_associated_module(next_version, module_name)
```

Comment:
This is to avoid analyzing `next_version` if it has a config error like so:
```
(Pdb) next_version.version_id
'mongodb_mongo_master_f31bc89f66632b2d521be2d076dc23f94ff663eb'
(Pdb) _get_associated_module(next_version, "enterprise")
*** tenacity.RetryError: RetryError[<Future at 0x7fcf61581590 state=finished raised HTTPError>]
(Pdb) next_version.build_by_variant(build.build_variant)
*** AttributeError: Unknown attribute build_variants_map
```